input/pagecontent/changes.md

```diff
@@ -14,4 +14,5 @@ This change log documents the significant updates and resolutions implemented fr
   - made ServiceRequest.encounter mandatory (1..1) [FHIR-47008](https://jira.hl7.org/browse/FHIR-47008)
   - added Must Support to ServiceRequest.priority [FHIR-46939](https://jira.hl7.org/browse/FHIR-46939)
   - AU eRequesting ServiceRequest profile renamed to be AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
+  - added [Request status reason extension](https://hl7.org/fhir/extensions/StructureDefinition-request-statusReason.html) [FHIR-47098](https://jira.hl7.org/browse/FHIR-47098)
```

Comment:
Suggest changing to something like: 
- added Request status reason extension as a Must Support element with cardinality 0..1

Reply:
done

Reply:
Thank you, just one minor change to remove the hyperlink from the extension as we don't normally hyperlink from the change log

Reply:
Removed hyperlink on behalf of @johnscarter.